core/css/tooltip.scss

```
@@ -25,7 +25,7 @@
     text-shadow: none;
     text-transform: none;
     white-space: normal;
-    word-break: normal;
+    word-break: break-all;
```

```suggestion
    word-break: normal;
    overflow-wrap: anywhere
```

This will still try first to use empty space before breaking words